guvnor-structure/guvnor-structure-api/src/main/java/org/guvnor/structure/repositories/PullRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.guvnor.structure.repositories;
+
+public interface PullRequest {
+
```

Comment:
@Salaboy @porcelli It's a good point, I agree that it should be added. But as this is not going to be the end of PRs I bet that we are going to refine many things. I've added this to my lists